### DISCUSSION TITLE
5244好像被运营商封了，求助！

### DISCUSSION THREAD

**Discussion by SfYangYang** [AlistGo/alist#4952] [Q&A]
刚搬到加拿大，用的是Bell宽带，结果发现Alist用不了。路由器管理后台没有关于防火墙的设置，打电话问了客服，也说没有防火墙，所以猜测是不是把5244端口封了。我换了443，也不行。

客服说我可以用端口转发功能。

对这块不熟悉。求助！

**Comment by kyon777** [comment]
这方面直接谷歌端口转发

**Comment by li-peifeng** [comment]
都叫你去设置端口映射了，你去路由器设置一下就行了，国外不会封端口

**Comment by SsuJojo** [comment]
你试一下lighttp能不能访问到，可以的话应该没问题的才对
